server/src/engineModel/EngineModel.ts

```diff
@@ -92,6 +93,27 @@ export class EngineModel {
                 this.completionProvider.addCompletionItem(method);
             });
         }
+
+        /* A class (or enum) with instances has the following format:
+        <class ident="EnumClassName">
+            <extends>yseop.lang.Enum</extends>
+            <instances count="2">
+                <item ident="EnumInstance_1"></item>
+                <item ident="EnumInstance_1"></item>
+            </instances>
+        </class>
+        */
+        if (!!yclass.instances && !!yclass.instances[0].item) {
+            // The current class has already existing instances.
+            // Iterate over each instances items.
+            yclass.instances[0].item.forEach((_instance) => {
+                const instance =
+                    ymlClass.extends.indexOf('yseop.lang.Enum') !== -1
```

Comment:
So it can appear anywhere? What if someone defines something like

`foo.bar.yseop.lang.Enum`

or even

`fooyseop.lang.Enumbar`

?

Reply:
`extends` is an Array of string.

Reply:
Omg I somehow missed the “.extends” part and thought that was the class name itself you were rumaging through. Sorry.